### DISCUSSION TITLE
Folody theme

### DISCUSSION THREAD

**Discussion by phatdev12** [JanDeDobbeleer/oh-my-posh#3456] [Themes]
**Hello everyone!**

Today I want to share with you about the theme that my team developed. It's also simple and nothing fancy.
A simple yet quite complete modern terminal theme developed by Folody.
At a glance, it is a terminal theme inspired by other themes but it still has certain characteristics.
I hope you have a good experience with this theme.

![image 1](https://user-images.githubusercontent.com/81029660/216805296-46f8ed5d-d510-4f9d-a929-d1eacf14a5a7.png)

**Comment by hanyuko** [comment]
this is MacOS theme right?
